tools/socktap/autotalks_link.cpp

```diff
@@ -0,0 +1,47 @@
+#include "autotalks_link.hpp"
+#include "autotalks.hpp"
+#include <vanetza/net/osi_layer.hpp>
+
+const unsigned SendBufferSize = 1000;
```

Comment:
The MTU is greater than 1000 bytes. I suggest allocating 2000 bytes on the stack to be on the safe side.